pom.xml

```diff
@@ -39,6 +39,11 @@
                 <artifactId>accession-commons-monotonic-generator-jpa</artifactId>
                 <version>0.4</version>
             </dependency>
+            <dependency>
+                <groupId>uk.ac.ebi.eva</groupId>
+                <artifactId>variation-commons-core</artifactId>
+                <version>0.6-SNAPSHOT</version>
```

Comment:
We can use variation-commons 0.6 final now :smiley:

Reply:
🙌